lib/src/main/java/org/team100/lib/motion/drivetrain/kinodynamics/SwerveKinodynamicsFactory.java

```diff
@@ -35,7 +35,7 @@ public static SwerveKinodynamics get() {
                 // TODO: make tests specify kinodynamics instead.
                 return new SwerveKinodynamics(4, 4, 4, 13, 20 * Math.PI, 0.5, 0.5, 0.3);
             case BETA_BOT:
-                return new SwerveKinodynamics(5, 5, 7, 13, 20 * Math.PI, 0.491, 0.765, 0.3);
+                return new SwerveKinodynamics(5, 5, 7, 13, 20 * Math.PI, 0.6096, 0.6096, 0.3);
```

Comment:
i think you're using the frame size here, 24 inches = 61 cm, not the wheelbase, 19 inches = 48 cm?